stories/atoms/components/icon/icon.stories.mdx

```diff
@@ -119,7 +122,7 @@ the **name** of the icon you want as the property value.
 
 ## Size
 
-The **size** is squared and can be adjusted by passing a number value.  
+The **size** is squared and can be adjusted by passing a number value.<br />
```

Comment:
Mmmh, I guess this change should be part of the future `story` contributing guidelines that will decide if we use `markdown` space syntax or HTML `<br>` tag to add a line break.
@ccamel what do you think?

Reply:
I agree, let's do it 👌

Reply:
ok I will fix my formatter and then remove the `<br />` tags